conference/DS/iccd.yml

```diff
@@ -14,3 +14,13 @@
       timezone: AoE
       date: November 6-8, 2023
       place: Washington DC, USA
+    - year: 2024
+      id: iccd24
+      link: https://www.iccd-conf.com/Call_for_papers.html
+      timeline:
+        - abstract_deadline: '2024-05-20 23:59:00'
+          deadline: '2023-05-20 23:59:00'
```

Comment:
seems to be 2024

Reply:
Yes, sorry, you are right, I will modify them. Thank you